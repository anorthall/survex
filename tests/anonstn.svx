; pos=yes warn=0 error=0
*fix 1 0 0 0
1 . 2.00 000 0
1 ... 10 - down
