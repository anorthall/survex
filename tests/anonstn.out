
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 3 survey stations, joined by 2 legs.
There are 0 loops.
Total length of survey legs =   10.00m (  10.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =   10.00m
Vertical range = 10.00m (from 1 at 0.00m to anonymous station at -10.00m)
Vertical range = 0.00m (from 1 at 0.00m to 1 at 0.00m)
North-South range = 2.00m (from anonymous station at 2.00m to 1 at 0.00m)
North-South range = 0.00m (from 1 at 0.00m to 1 at 0.00m)
East-West range = 0.00m (from 1 at 0.00m to 1 at 0.00m)
   2 1-nodes.
   1 2-node.
